projects/knora-ui/src/lib/viewer/values/base-value.component.ts

```diff
@@ -25,6 +25,8 @@ export abstract class BaseValueComponent {
 
   form: FormGroup;
 
+  abstract customValidators: ValidatorFn[];
```

Comment:
@mdelez I added the abstract member `customValidators` that has to be implemented for each value type or set to an empty array if not needed.

Reply:
Very nice :)